apps/core/templates/core/blocks/alert.html

```diff
@@ -1,14 +1,12 @@
 {% load wagtailcore_tags %}
-<div class="alert my-4">
-    <div class="alert__type-{{self.alert_type}} p-2 px-3 text-white fw-bold rounded-top d-flex align-items-center">
-        <div class="me-2 d-flex align-items-center">
+<div class="alert alert--{{ self.alert_type|lower }}">
+    <div class="alert__header">
+        <div class="alert__icon">
             {% include self.icon %}
         </div>
-        {{self.alert_type}}
+        <h2 class="alert__heading">{{ self.alert_type }}</h2>
```

Comment:
I don’t think this is appropriate as a heading, because those alerts can technically be part of an existing section, and because if we had multiple alerts on the page they’d all end up having the same heading.

Filed in #150.